File: remove_bom_js.py
import re

with open(r'd:\LEMP\eBMR\resources\views\pages\category\intermediate\dataTable.blade.php', 'r', encoding='utf-8') as f:
    lines = f.readlines()

new_lines = []
skip = False
for i, line in enumerate(lines):
    if "window.renderBOMRows = function(" in line:
        skip = True
    
    if skip and "</script>" in line:
        skip = False
        new_lines.append(line)
        continue
        
    if not skip:
        new_lines.append(line)

with open(r'd:\LEMP\eBMR\resources\views\pages\category\intermediate\dataTable.blade.php', 'w', encoding='utf-8') as f:
    f.writelines(new_lines)

File: remove_bom_js.php
<?php
$file = 'd:/LEMP/eBMR/resources/views/pages/category/intermediate/dataTable.blade.php';
$content = file_get_contents($file);
$content = preg_replace('/window\.renderBOMRows\s*=\s*function.*?\}\);(?=\s*<\/script>)/s', '', $content);
$content = preg_replace('/\/\/ Sync Summernote div content.*?\}\);/s', '', $content);
file_put_contents($file, $content);
